src/inventory.cpp

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2020 YADRO
+
+#include "inventory.hpp"
+
+#include <config.hpp>
+
+/**
+ * @brief Construct item name from its path.
+ *
+ * @param[in] path - D-Bus path of the item
+ *
+ * @return inventory item name
+ */
+static std::string nameFromPath(const std::string& path)
+{
+    size_t lastSlash = path.find_last_of('/');
```

Comment:
Fixed

> Just wondering why `find_last_of` and not `rfind` ? They look to be completely identical in this particular case.

I agree, fixed

> Also, I guess `lastSlash` could be `auto`.

Do you want to replace size_t with auto? Why?

Reply:
Why does `auto` exist at all? ;) I don't insist. Just though that since there is `auto`, you don't need to declare the variable based on your assumption (ok, knowledge) regarding the return type of the function you're going to assign to that variable.

Reply:
`auto` is meaningful if you don't care about types or you just lazy :).
For me, it is easier to read code with static types.